docs/storage.md

```diff
@@ -0,0 +1,97 @@
+
+Typically you will only need to tune the first two, depending on where you want the storage to write its data, and for how long you want to keep it.
+
+On average, Prometheus uses only around 1-2 bytes per sample. Thus, to plan the capacity of a Prometheus server, you can use the rough formula:
+
+```
+needed_disk_space = retention_time_seconds * ingested_samples_per_second * bytes_per_sample
+```
+
+To tune the rate of ingested samples per second, you can either reduce the number of time series you scrape (fewer targets or fewer series per target), or you can reduce the scrape rate. However, reducing the number of series is likely more effective.
+
+If your local storage becomes corrupted for whatever reason, your best bet is to shut down Prometheus and remove the entire storage directory. However, you can also try removing individual block directories to resolve the problem. This means losing a time window of around two hours worth of data per block directory. Again, Prometheus's local storage is not meant as a durable long-term storage.
+
+## Remote storage integrations
+
+As outlined above, Prometheus's local storage is limited in its scalability and durability. Instead of trying to solve long-term storage in Prometheus itself, Prometheus has a set of interfaces that allow integrating with remote long-term storage systems.
+
+### Overview
+
+Prometheus integrates with remote storage systems in two ways:
+
+* Prometheus can write all samples that it ingests to a remote URL in a standardized format.
+* Prometheus can read (back) sample data from a remote URL in a standardized format.
+
+![Remote read and write architecture](images/remote_integrations.png)
+
+The read and write protocols both use a snappy-compressed protocol buffer encoding over HTTP. The protocols are still marked experimental and may change to use gRPC over HTTP/2 in the future, when all network hops between Prometheus and the long-term storage can safely be assumed to support HTTP/2.
+
+For details on configuring remote storage integrations in Prometheus, see the [remote write](configuration/configuration.md#remote_write) and [remote read](configuration/configuration.md#remote_read) sections of the Prometheus configuration documentation.
+
+For details on the request and response messages, see the [remote storage protocol buffer definitions](https://github.com/prometheus/prometheus/blob/master/storage/remote/remote.proto).
+
+Note that on the read path, Prometheus only fetches raw series data for a set of label selectors and time ranges from the remote end. All PromQL evaluation on the raw data still happens in Prometheus itself. This means that remote read queries have some scalability limit, since all necessary data needs to be loaded into the querying Prometheus server first and then processed there. However, supporting fully distributed evaluation of PromQL was deemed infeasible for the time being.
+
+### Existing integrations
```

Comment:
These are already listed on the Integrations page

Reply:
Ah damn, somehow didn't find that earlier, probably because I only grepped for Crate, which I was sure you would have added if such a page existed ;) Will instead point to that page.

Reply:
When I created it the Crate stuff wasn't sufficiently launched to list it yet.